lte/gateway/c/core/oai/test/mock_tasks/mme_app_mock.cpp

```diff
@@ -163,6 +163,14 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
     } break;
 
     case S1AP_PATH_SWITCH_REQUEST: {
+      mme_app_handler_->mme_app_handle_path_switch_request();
+      e_rab_to_be_switched_in_downlink_list_t* e_rab_to_be_switched =
+          &S1AP_PATH_SWITCH_REQUEST(received_message_p)
+               .e_rab_to_be_switched_dl_list;
+      for (int i = 0; i < e_rab_to_be_switched->no_of_items; i++) {
+        bdestroy_wrapper(
+            &e_rab_to_be_switched->item[i].transport_layer_address);
```

Comment:
This should be done in `itti_free_msg_content`

Reply:
it was being done on the handlers so that's why I replicated it on the mock thread, I moved it to the `itti_free_msg_content` to avoid replication